ArduPlane/commands_logic.cpp

```diff
@@ -252,12 +252,22 @@ bool Plane::verify_command(const AP_Mission::Mission_Command& cmd)        // Ret
             return landing.verify_abort_landing(prev_WP_loc, next_WP_loc, current_loc, auto_state.takeoff_altitude_rel_cm, throttle_suppressed);
 
         } else {
+            float height = height_above_target();
+
             // use rangefinder to correct if possible
-            float height = height_above_target() - rangefinder_correction();
+            if (landing.get_rangefinder_holdoff_distance() <= 0 ||  // rangefinder holdoff is disabled (never hold off, always use rtanegfinder)
+                auto_state.wp_proportion >= 1 ||                    // or we have reached the land waypoint
+                landing.is_flaring() ||                             // or we are flaring
```

Comment:
I'm not sure its a good idea to change the altitude once the flare has started. I don't think it will do anything in anycase.

Reply:
Why not?